sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +150,159 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun showAnnotations(
+    annotations: List<View>,
+    edgeInsets: EdgeInsets?,
+    bearing: Double?,
+    pitch: Double?,
+    animationDurationMs: Long
+  ) {
+    if (annotations.isNotEmpty()) {
+      val viewAnnotationOptions = annotations.mapNotNull {
+        getViewAnnotationOptionsByView(it)
+      }
+      val coordinateBounds = coordinateBoundsFromAnnotations(viewAnnotationOptions)
```

Comment:
what if annotation is `View.INVISIBLE` or `View.GONE`? not sure it should participate in calculations then

Reply:
I am testing that. will add condition for that too